src/components/calcite-shell-panel/calcite-shell-panel.tsx

```diff
@@ -50,13 +63,56 @@ export class CalciteShellPanel {
    */
   @Prop({ reflect: true }) position: Position;
 
+  /** Accessible label for resize separator.
```

Comment:
Nitpick: Can you start the doc text in the next line?

![200](https://user-images.githubusercontent.com/197440/142698606-420654bc-dcba-447e-adf6-9aba466993f7.gif)